zappa/core.py

```diff
@@ -2178,17 +2178,20 @@ def update_domain_base_path_mapping(self, domain_name, lambda_name, stage, base_
             return
         base_path_mappings = self.apigateway_client.get_base_path_mappings(domainName=domain_name)
         found = False
-        for base_path_mapping in base_path_mappings['items']:
-            if base_path_mapping['restApiId'] == api_id and base_path_mapping['stage'] == stage:
-                found = True
-                if base_path_mapping['basePath'] != base_path:
-                    self.apigateway_client.update_base_path_mapping(domainName=domain_name,
-                                                                    basePath=base_path_mapping['basePath'],
-                                                                    patchOperations=[
-                                                                        {"op" : "replace",
-                                                                         "path" : "/basePath",
-                                                                         "value" : '' if base_path is None else base_path}
-                                                                    ])
+        try:
+            for base_path_mapping in base_path_mappings['items']:
```

Comment:
Why not just use __dict.get()__?  Like __for base_path_mapping in base_path_mappings.get('items', [])__

Reply:
You are right - that is better. Implementing.

Reply:
Do I need to get something to improve coverage in order to get this accepted?